src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+const listItems = [...tree.querySelectorAll('li')];
+
+listItems.forEach((elem) => {
+  const firstText = elem.firstChild;
+
+  if (firstText.nodeType === 3) {
+    const span = document.createElement('span');
+
+    span.textContent = `${firstText.textContent}`;
+
+    elem.replaceChild(span, firstText);
+  }
+});
+
+// eslint-disable-next-line no-shadow
+function clickOnSpan(event) {
+  const spanTarget = event.target.closest('span');
+
+  if (spanTarget) {
+    const list = spanTarget.nextElementSibling;
```

Comment:
Ensure that `spanTarget.nextElementSibling` is not `null` before proceeding. Add a check to handle cases where there is no sibling element.